lua/neogit/lib/git/cli.lua

```diff
@@ -701,6 +724,9 @@ local function new_builder(subcommand)
       table.insert(cmd, 1, state.prefix)
     end
 
+    -- Disable the pager so that the commands don't stop and wait for pagination
+    cmd = util.merge({ "git", "--no-pager", "-c", "color.ui=always", "--no-optional-locks", subcommand }, cmd)
```

Comment:
This got moved down to fix a bug: `prefix` would insert at index 1, which was before the string `git`.